ci/scripts/towncrier_automation.py

```diff
@@ -16,6 +17,14 @@ class Args(argparse.Namespace):
     dry_run: bool
 
 
+class NoPatchReleaseOnMainError(Exception):
+    pass
+
+
+class NoMinorMajorReleaseOffMainError(Exception):
+    pass
```

Comment:
Hmm, that made me think. Ideally it’d look like this right?

```console
❯ uv run ci/scripts/towncrier_automation.py 1.11.4
usage: towncrier-automation [-h] [--dry-run] version
towncrier-automation: error: argument version: 1.11.4 is a patch release, but you are trying to release from a non-patch release branch: ig/no_patch_release_on_main_error.
```

I filed #1770 to make it that way